### ISSUE TITLE
Adjustable Size for swatches triger (vue-swatches__trigger)

### ISSUE THREAD

**Issue by saloustrosm** [saintplay/vue-swatches#1]
Really nice module, it would be great if we could change the swatches trigger size (now is 42px) and calculate automatically the radius

**Comment by saintplay** [comment]
Thank you for submitting an issue! Right now modifying via css is not possible (unless you use **!important**)  
Because a lot of styles are calculated and applied as inline styles.

I will make `trigger-style`, `wrapper-style`, and `swatch-style`props for proper customization

**Comment by saintplay** [comment]
```html
<swatches
   v-model="color"
   :trigger-style="{ width: '30px', height: '30px' }"
/>
```

**Comment by saloustrosm** [comment]
That is great and exactly what i wanted so thank you very much.

Could you please compile it to dist in order to work just with "npm install"?

Thanks again.

**Comment by saintplay** [comment]
Ups my bad!
fixed in 1.0.0-rc.7!

**Comment by saloustrosm** [comment]
Great, is working really nice now

THANKS
